Tutorial.md

```diff
@@ -11,477 +10,251 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                        "votes": 2048
+                    }, {
+                        "choice": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "choice": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "choice": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
             }
+```
 
-    + Response 201 (application/hal+json)
-
-        [Gist][]
-
-At this point you should be familiar with everything in this listing except the definition and discussion of the query parameter *"since"*:
+#### Response Without a Body
 
-<a name="query_parameters"></a>
-#### Query Parameters
+This resource has a delete action. The server will return a 204 response without a body.
 
-    ## Gists Collection [/gists{?since}]
-
-As with the URI parameters the Query parameters are defined in the [URI Template][] format. The difference here is that a query parameter name is preceded by the *question mark* and it is always at the end of the URI Template.
-
-> **NOTE:** To define multiple query parameters simply comma-separate their names. For example `{?month,year}`.
-
-#### Action-specific Query Parameters
-
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
-
-Often a resource query parameter is specific to just one of the resource actions. In this case you can discuss it in the relevant action only. No matter where you discuss a query parameter it follows exactly the same syntax as with [URI Parameters][].
-
-## Subsequent Resource
-The last missing piece in the expected functionality is the Star Resource and its actions. Utilizing everything we have learned so far the Star Resource and its actions are defined as follows:
-
-    ## Star [/gists/{id}/star]
-    Star resource represents a Gist starred status.
-
-    The Star resource has the following attribute:
-
-    - starred
-
-
-    + Parameters
-
-        + id (string) ... ID of the gist in the form of a hash
-
-    + Model (application/hal+json)
-
-        HAL+JSON representation of Star Resource.
-
-        + Headers
-
-                Link: <http:/api.gistfox.com/gists/42/star>;rel="self"
-
-        + Body
-
-                {
-                    "_links": {
-                        "self": { "href": "/gists/42/star" },
-                    },
-                    "starred": true
-                }
-
-    ### Star a Gist [PUT]
-    + Response 204
-
-    ### Unstar a Gist [DELETE]
-    + Response 204
-
-    ### Check if a Gist is Starred [GET]
-    + Response 200
-
-        [Star][]
+```
+### Delete [DELETE]
++ Response 204
```

Comment:
Minor: 

For consistency I would say you want a newline between header and the list item